src/config.js

```diff
@@ -217,6 +217,7 @@ global.config = {
 
   myRoom: {
     underAttackMinAttackTimer: 50,
+    leastSpawnsToRebuildStructureSpawn: 1,
```

Comment:
so should this be 1 or 2

Reply:
I would use 2 for myself, but there be a unit test here requires MUST relocate spawn as soon as possible, otherwise would fail.
So I have to set it default 1.

Reply:
can you point me to it, can't find it?
![image](https://user-images.githubusercontent.com/2712470/189446155-9769e9ec-ff4e-4e24-9aff-3ce44c067ade.png)

Reply:
> can you point me to it, can't find it? ![image](https://user-images.githubusercontent.com/2712470/189446155-9769e9ec-ff4e-4e24-9aff-3ce44c067ade.png)

I make it to 2 and commit, then we can see it from circleci result...

Reply:
@mschultheiss83 see https://app.circleci.com/pipelines/github/TooAngel/screeps/237/workflows/d2d2ba27-f834-4b4b-82f4-456636e8e5d8/jobs/2532